js-miniapp-sdk/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## CHANGELOG
 
-### 1.6.0 (2020-12-18)
+### 1.7.0 (In progress)
```

Comment:
1.6.0 was already released, so you'll need to make a new section for 1.7.0.

Reply:
Did make a new section for  1.7.0